config/common/traces-limits-v2.toml

```diff
@@ -10,34 +10,34 @@ BLOCK_HASH = 512
 EC_DATA = 262144
 EUC = 65536
 EXP = 8192
-EXT = 65536
+EXT = 1048576
 GAS = 65536
 HUB = 2097152
 LOG_DATA = 65536
 LOG_INFO = 4096
-MMIO = 2097152
+MMIO = 4194304
 MMU = 4194304
 MOD = 131072
 MUL = 65536
 MXP = 524288
 OOB = 262144
 RLP_ADDR = 4096
 RLP_TXN = 131072
-RLP_TXN_RCPT = 32768
-ROM = 8388608
+RLP_TXN_RCPT = 65536
+ROM = 4194304
 ROM_LEX = 1024
 SHAKIRA_DATA = 32768
 SHF = 65536
-STP = 32768
-TRM = 8192
+STP = 16384
+TRM = 32768
 TXN_DATA = 8192
 WCP = 262144
 #
-# Reference table limits
+# Reference table limits, set to UInt.MAX_VALUE
 #
-BIN_REFERENCE_TABLE = 196864
-SHF_REFERENCE_TABLE = 4096
-INSTRUCTION_DECODER = 512
+BIN_REFERENCE_TABLE = 4294967295
```

Comment:
In prod envs there were issues with this value used in Besu, because the Int is signed there. We had to change it to `2147483647` instead

Reply:
Actually it doesn't matter here. It would be important only for `traces-limits-besu-v2.toml` file. This one is for Coordinator and it's Ok to have it set to `4294967295` here